labs/coding-101-rest-basics-ga/6.md

```diff
@@ -40,12 +40,12 @@ To get the entire list of hosts, your request will be like this:
 * **URL** - `https://{APIC-EM-Server}/api/v1/host`
 	* Insert the IP address or hostname of the APIC-EM controller.
 * **Headers**
-	* For authentication, add the service ticket that was created earlier. On the left side of the header, add the text 'X-Auth-Token'. On the right side, add the service ticket value.
-	* When using GET, a content header is not required. However, it's a good practice to specify the content in the header anyway as  'Content-Type' 'application/json'
+	* For authentication, add the service ticket that was created earlier. On the left side of the header, add the text 'X-Auth-Token'. On the right side, add the service ticket value. See screenshot on next page for information on how to set this up in Postman.
+	* When using GET, a content header is not required. However, it's a good practice to specify the content in the header anyway as  'Content-Type' 'application/json'.
 * **Authentication**
 	* Handled in the header.
 * **Body**
-	 * Not required for this request
+	 * Not required for this request.
 
 ![](/posts/files/coding-101-rest-basics-ga/assets/images/apic-emHostRequest.png "Get Hosts")
```

Comment:
Okay, so this is the reference for the screenshot - going to `/posts/files/lab-name/assets/images/apic-emHostRequest.png` but the actual file storage is in this repo in `lab-name/assets/images/apic-emHostRequest.png`. So you can replace the file here in the repo in this or another Pull Request. Once you copy over the .png file, git should recognized that the file changed. Does that make sense? I can do an example if that helps, just let me know.